api/envoy/extensions/filters/http/aws_request_signing/v3/aws_request_signing.proto

```diff
@@ -87,6 +96,14 @@ message AwsRequestSigning {
 
   // Optional Signing algorithm specifier, either ``AWS_SIGV4`` or ``AWS_SIGV4A``, defaulting to ``AWS_SIGV4``.
   SigningAlgorithm signing_algorithm = 6;
+
+  // Use the query string to store output of SigV4 or SigV4A calculation, rather than HTTP headers. The ``Authorization`` header will not be modified if ``query_string``
+  // is configured
```

Comment:
I'm fine with the wording but I do think it would be better to start the sentence with "If set, use the query string" ... 
That makes it clearer that this field is meant to trigger a certain behavior iff it is set.  

Also, if you can please end the sentence with a period (i.e. `.`).  

Thanks!